club/style.css

```
@@ -105,14 +105,56 @@ GNU General Public License for more details.
 	border: 1px solid var(--wp--preset--color--foreground);
 }
 
-/* This is necesary to make the post list item the same height when the post has or lacks a featured image */
+/* 
+ * Styles for the post list pattern
+ */
+
 .club-post-list-pattern .wp-block-post-title {
+	/* This is necesary to make the post list item the same height when the post has or lacks a featured image */
 	min-height: 60px; /* 60px is the height of the featured image used */
     display: flex;
     justify-content: center;
     flex-direction: column;
 }
 
+.club-post-list-pattern .wp-block-post-date {
+	white-space: nowrap;
+}
+
+.club-post-list-pattern .wp-block-read-more {
+	width: auto;
+	text-align: center;
+	word-break: keep-all;
+	width: 100%;
+	white-space: nowrap;
+}
```

Noticed this duplicate definition:

```suggestion
.club-post-list-pattern .wp-block-read-more {
	text-align: center;
	word-break: keep-all;
	width: 100%;
	white-space: nowrap;
}
```